shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/loader/dialect/OracleSchemaMetaDataLoader.java

```diff
@@ -66,17 +68,32 @@ public final class OracleSchemaMetaDataLoader implements DialectSchemaMetaDataLo
     
     private static final int IDENTITY_COLUMN_START_MINOR_VERSION = 1;
     
+    private static final int BATCH_SIZE = 1000;
```

Comment:
@cheese8 Can we run some tests to determine which batch size perform best for metadata load?

Reply:
The max what oracle supported is 1000 which ORA-01795 shows, and I think the less database queried the best performance get.

And `performance optimized for get connection` PRed to reuse the connection.